### DISCUSSION TITLE
Should prisma/generated be checked in to VCS?

### DISCUSSION THREAD

**Discussion by revmischa** [maoosi/prisma-appsync#24] [Q&A] [answered]
Should I ignore or check in `prisma/generated`?

**Comment by maoosi** [comment, marked as answer]
> Should I ignore or check in `prisma/generated`?

You can safely ignore it, as the `prisma/generated` directory will automatically be re-generated each time you run `npx prisma generate` and before each deploy (when using the cdk boilerplate).
